src/config/dynamic-theme-fixes.config

```diff
@@ -2868,6 +2868,15 @@ CSS
 
 ================================
 
+community.cloudflare.com
+
+CSS
+.visited {
+  background-color: #262626;
+}
```

Comment:
What about having a fix like:
```suggestion
:root {
    --darkreader-text--primary-medium: ${gray} !important;
}
```

Reply:
Yeah, could work. My eyes aren't the best so I find mine much easier to distinguish between the two. If you wish to merge your fix in though, I am happy with it. I can still keep my fix for just me

![image](https://user-images.githubusercontent.com/8492901/140605640-7833ba4a-3717-4a76-8cd5-6b7395b7b3d6.png)

Reply:
> If you wish to merge your fix in though

Yeah, the fixes config should stay as the original page(which also just modifies the text's color).